.gitignore

```diff
@@ -29,4 +29,5 @@ vendor/
 fastlane/report.xml
 google-services.json
 output-metadata.json
-service.properties
\ No newline at end of file
+service.properties
+wallet/schemas/*/*.json
```

Comment:
So we don't need these files in our repo?

I thought these files were needed for backward compatibility with older database versions.

Reply:
These files are actually generated on build. We don't need to keep them, and in some other branches, they were already excluded in .gitignore.

Right now our database is actually performing a destructive migration on every database version increment. That means everything is erased and re-created again. For keeping data when updating the database version, we need to write migrations.

Reply:
Re-added since we still might need them for testing migrations.